master/usage/configuration/ip-in-ip.md

```diff
@@ -0,0 +1,89 @@
+over the fabric and you are not peering your Calico BGP network with the rest of the
+network.
+
+Calico can be configured to use IP-in-IP encapsulation by enabling the IPIP option
+on the [IP pool resource]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool). 
+When enabled, Calico will use IP-in-IP encapsulation when routing packets *to*
+workload IPs falling in the IP pool range.
+
+An optional `mode` field toggles when IP-in-IP is used:
+-  A value of `always` is used when IP-in-IP is always required.  This is the default
+   value used if the `mode` field is omitted in the configuration.
+-  A value of `cross-subnet` is used when IP-in-IP is only required when routing 
+   between hosts on different subnets.
+   
+> **NOTE**
+>
+> Routing between Workloads and Host Endpoints running Calico is asymmetric when
+> IPIP is enabled, this causes traffic to be discarded between them.  To resolve 
+> this, enable nat-outgoing or use the IPIP tunnel address on the host
+> for the Host Endpoint IP address.
+
+### Configuring cross-subnet IP-in-IP
+
+The following `calicoctl` example will create or modify an IPv4 pool with 
+CIDR 192.168.0.0/16 using IPIP mode `cross-subnet`.
+
+```
+$ calicoctl apply -f - << EOF
+apiVersion: v1
+kind: ipPool
+metadata:
+  cidr: 192.168.0.0/16
+spec:
+  ipip:
+    enabled: true
+    mode: cross-subnet
+EOF
+```
+
+> **Note** 
+>
+> The `cross-subnet` mode option requires each Calico node to be configured
+> with the IP address and subnet of the host.  However, the subnet configuration
+> was only introduced in Calico v2.1.  If any nodes in your deployment were originally
+> created with an older version of Calico, or if you if you are unsure whether 
+> your deployment is configured correctly, follow the steps in 
+> [Upgrading from pre-v2.1](#upgrading-from-pre-v21) before enabling "cross-subnet" IPIP.
+
+#### Upgrading from pre-v2.1
+
+If you are planning to use cross-subnet IPIP, your entire deployment must be running with
+Calico v2.1 or higher.  See [releases page]({{site.baseurl}}/{{page.version}}/releases) 
+for details on the component versions for each release.
+
+Upgrade your deployment to use the latest Calico versions - the process for this
+will be dependent on your orchestration system (if using one). 
+
+Prior to Calico v2.1, the subnet information was not detected and stored on the 
+node configuration.  Thus, if you have calico/node instances that were deployed 
+prior to v2.1, the node configuration may need updating to fix the host subnet.
+The subnet configuration must be set correctly for each node before `cross-subnet`
+IPIP mode is enabled.
+
+You can verify which of your nodes is correctly configured using calicoctl.
+
+Run `calicoctl get nodes --output=wide` to check the configuration.  e.g.
+
+```
+$ calicoctl get nodes --output=wide
+NAME    ASN       IPV4           IPV6   
+node1   (64512)   10.0.2.15/24          
```

Comment:
Does this need to be updated in the examples in  http://docs.projectcalico.org/master/reference/calicoctl/resources/node too?